manager/integration/tests/test_basic.py

```diff
@@ -2952,12 +2954,12 @@ def test_backup_lock_creation_during_deletion(set_random_backupstore, client, co
     _, b1 = common.find_backup(client, std_volume_name, snap1.name)
 
     write_pod_volume_random_data(core_api, std_pod_name,
-                                 "/data/test2", DATA_SIZE_IN_MB_1)
-
-    backup_volume.backupDelete(name=b1.name)
+                                 "/data/test2", DATA_SIZE_IN_MB_2)
 
     snap2 = create_snapshot(client, std_volume_name)
 
+    backup_volume.backupDelete(name=b1.name)
+
```

Comment:
Create second snapshot first, then consequently delete backup 1 and trying to create backup 2